### PR TITLE
Add pxc-package-testing parallel job

### PR DESCRIPTION
**Pull request by panchal-yash**
Add pxc package-testing parallel execution jenkins job

### CODE REVIEW

File: pxc/jenkins/pxc-package-testing-parallel.groovy [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            string(name: "install_repo", value: params.install_repo),
+            string(name: "node_to_test", value: node_to_test)
+        ],
+        propagate: true,
+        wait: true
+    )
+}
+
+pipeline {
+    agent none
```
Comment (eleo007) [inline pxc/jenkins/pxc-package-testing-parallel.groovy:38]: Add agent - for example docker. This is best practice to run on some agent.

File: pxc/jenkins/pxc-package-testing-parallel.groovy [hunk 2]
```diff
@@ -0,0 +1,147 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+List all_nodes = [
+                'ubuntu-focal',
```
Comment (eleo007) [inline pxc/jenkins/pxc-package-testing-parallel.groovy:7]: Add 'ubuntu-jammy' as it is in the list of wip

**Review by eleo007 — CHANGES_REQUESTED** [review]

**Review by eleo007 — DISMISSED** [review]

**Review by eleo007 — CHANGES_REQUESTED** [review]
PXC 57 tests have 'pxc57_repo' option. PXC57 packages are stored in 2 repos: original and pxc57. So we need to move this functionality here - to select in from what repo to install packages for test.

File: pxc/jenkins/pxc-package-testing.groovy [hunk 3]
```diff
@@ -40,16 +40,17 @@ void runMoleculeAction(String action, String version, String scenario) {
         sh """
             source venv/bin/activate
             export MOLECULE_DEBUG=0
-            export install_repo=${params.INSTALL_REPO}
+            export install_repo=${params.install_repo}
+            export pxc57repo=${params.pxc57_repo}
```
Comment (eleo007) [inline pxc/jenkins/pxc-package-testing.groovy:44]: We need this var only for pxc57 tests so on old jenkins we export it only for pxc57, please do here the same.

Comment (panchal-yash) [reply pxc/jenkins/pxc-package-testing.groovy]: Hi,

Added a conditional block for setting pxc57repo variable.

**Review by eleo007 — APPROVED** [review]

**Pull request merged**